docs/modules/ROOT/pages/metrics.adoc

```diff
@@ -0,0 +1,1678 @@
+
+|`cluster.clock.localClockTime`
+|ms
+|this member's local clock timestamp
+
+|`cluster.clock.maxClusterTimeDiff`
+|ms
+|the maximum observed cluster time difference
+
+|`cluster.connection.closedTime`
+|count
+|the connection close time for this connection
+
+|`cluster.connection.connectionId`
+|count
+|the connection id for this client connection
+
+|`cluster.connection.eventHandlerCount`
+|count
+|the number of event handler for this client connection
+
+|`cluster.heartbeat.lastHeartbeat`
+|ms
+|the last timestamp of sending heartbeats to the other known cluster members
+
+|`cluster.size`
+|count
+|the size of the cluster
+
+|`cp.atomiclong.value`
+|count
+|the value of this AtomicLong
+
+|`cp.countdownlatch.count`
+|count
+|the initial count of the `ICountDownLatch`
+
+|`cp.countdownlatch.remaining`
+|count
+|the remaining number of expected count-downs
+
+|`cp.countdownlatch.round`
+|count
+|the round number of the `ICountDownLatch`, each time `ICountDownLatch` is initialized with a new count after it downs to zero, a new round begins
+
+|`cp.lock.acquireLimit`
+|count
+|the maximum number of reentrant acquires of this `FencedLock`
+
+|`cp.lock.lockCount`
+|count
+|the number that shows how many times this `FencedLock` acquired
+
+|`cp.lock.owner`
+|count
+|the address of the `FencedLock` owner
+
+|`cp.lock.ownerSessionId`
+|count
+|the sessionId of the `FencedLock` owner
+
+|`cp.semaphore.available`
+|count
+|the number of the remaining available permits
+
+|`cp.semaphore.initialized`
+|count
+|a boolean state, which shows whether semaphore is initialized with a value or not. In exposed metrics it will be 0 when semaphore is not initialized, a positive value otherwise
+
+|`cp.session.creationTime`
+|ms
+|the creation time of this session
+
+|`cp.session.endpoint`
+|
+|the address of the endpoint which session belongs to
+
+|`cp.session.endpointType`
+|
+|the type of the endpoint; either SERVER or CLIENT
+
+|`cp.session.expirationTime`
+|ms
+|the expiration time of the session
+
+|`cp.session.version`
+|count
+|a boolean state, which shows whether semaphore is initialized with a value or not. In exposed metrics it will be 0 when semaphore is not initialized, a positive value otherwise
+
+|`event.eventQueueSize`
+|count
+|the total number of events pending to be processed
+
+|`event.eventsProcessed`
+|count
+|the total number of processed events
+
+|`event.listenerCount`
+|count
+|the number of subscribed listeners for the specified service
+
+|`event.publicationCount`
+|count
+|the number of published events for the specified service
+
+|`event.queueCapacity`
+|count
+|the queue capacity of the executor processing the events. This capacity is shared for all events
+
+|`event.rejectedCount`
+|count
+|the number of rejected events. If the event is not accepted to the executor in `hazelcast.event.queue.timeout.millis`(see xref:system-properties.adoc[]), it will be rejected and not processed
```

Comment:
Number of the rejected events; if the event is not accepted by the executor in `hazelcast .....

(It'd be good to place an anchor on the related system property to forward the reader directly to the description of it in the System Properties appendix, I'll take care of it).